source/braille.py

```diff
@@ -1769,7 +1769,7 @@ def __init__(self, handler):
 		#: The translated braille representation of the entire buffer.
 		#: @type: [int, ...]
 		self.brailleCells = []
-		self._windowRowBufferOffsets: list[tuple[int, int]] = [(0, 1)]
+		self._windowRowBufferOffsets: list[tuple[int, int]] = [(0, 0)]
```

Comment:
there are appears to be merge conflicts here

Reply:
I think this is fixed now.